utils/depth.go

```diff
@@ -0,0 +1,36 @@
+package utils
+
+import "image/color"
+
+func ColorModel(depth int) color.Model {
```

Comment:
Is this file still used?

Reply:
I forgot to remove it. I will do this.

Reply:
File removed in [commit 1c82249](https://github.com/zhaori96/barcode/commit/1c8224977e1efd6f585a2fb6a20809b6af6b5dd8)